src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/UnusedImportsCheck.java

```diff
@@ -174,6 +178,19 @@ else if (ast.getType() == TokenTypes.STATIC_IMPORT) {
     }
 
     /**
+     * Checks whether an import is unused.
+     * @param imp import ident.
+     * @return true if an import is unused.
+     */
+    private boolean isUnusedImport(FullIdent imp) {
```

Comment:
please make argument as String and name it `imprt`, 

I read `imp` as `implementation` for some reason.